backend/src/routes/aggregators/applications.ts

```diff
@@ -210,6 +280,50 @@ export function filterApplications(filters: FilterSelections, items: ICompressed
   return items
 }
 
+export function getStatusFilterKey(item: ICompressedResource, index: AppColumns) {
+  const score = (item.transform[TransformColumns.scores] as ApplicationScoresMap[])[0][index]
+  switch (index) {
+    case AppColumns.health:
+      return score < 100 ? 'Healthy' : 'Unhealthy'
+    case AppColumns.synced:
+      return score < 100 ? 'Synced' : 'OutOfSync'
+    case AppColumns.deployed:
+      return score < 100 ? 'Deployed' : 'Not Deployed'
```

Comment:
I think the way the scoring works with powers of 10, if I have more than 100 healthy/synced/deployed items... things will look bad, right?